gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=http\://services.gradle.org/distributions-snapshots/gradle-5.1-20181029034539+0000-bin.zip
+distributionUrl=https\://services.gradle.org/distributions-snapshots/gradle-5.1-20181128000036+0000-bin.zip
```

Comment:
Interesting, why did we change to `https` here, and why were we using `http` before?

Reply:
I used the wrapper task. Looks like @eskatos switched to `http`.

@eskatos What were the reasons to use `http` instead of `https` when updating the wrapper?

Reply:
@wolfs I think I edited that file instead of using the task because I had the version in by buffer, bad idea, I must have made a mistake while editing 🤦‍♂️